### DISCUSSION TITLE
404 index.html (GitHub pages)

### DISCUSSION THREAD

**Discussion by jakeisadev** [community/community#53328] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have checked the index.html file spelling and repo contents in general, I cannot for the life of me figure out why I am still receiving the following error on my pages site: 

"404
File not found

The site configured at this address does not contain the requested file.

If this is your site, make sure that the filename case matches the URL.
For root URLs (like http://example.com/) you must provide an index.html file.

[Read the full documentation](https://help.github.com/pages/) for more information about using GitHub Pages."

-----

Here is the pages link: https://jakeisadev.github.io/hairbyladyb/

Here is the link to my repo: https://github.com/jakeisadev/hairbyladyb

----

I've attempted to search through other discussions but it seems to not help my current situation. That, or maybe i'm misunderstanding some of the solutions provided. Thank you for your attention.

**Comment by airtower-luna** [comment, marked as answer]
The problem is that your `index.html` (and everything else) isn't at the root of your repository, it's in a `hairbyladyb-master/` directory. That's why your index shows up here: https://jakeisadev.github.io/hairbyladyb/hairbyladyb-master/

Move your files out of that directory and it'll work. :slightly_smiling_face:

**Reply by jakeisadev** [reply]
Excuse me while I sit outside in silence contemplating why I didn't notice that... Lol

Thank you!

**Reply by airtower-luna** [reply]
Glad I could help! Please don't forget to mark the answer. :slightly_smiling_face:

**Comment by konli90** [comment]
@airtower-luna I have the same scenario. I have tried to remove the files to root of the folder still getting 404.

Now it has worked after deleted the repository and starting all over again. The issue was that i was cloning the repository with .git at the end

**Reply by tuves** [reply]
@konli90 I've edited your post as it contained PII, which we do not allow in the GitHub Community. Please respect other community members by asking communicating here in an open forum, rather than asking folks to contact you directly.

**Reply by konli90** [reply]
Noted
